generator/sbpg/targets/resources/sbp_messages_mod.rs

```diff
@@ -27,6 +27,7 @@ use crate::serialize::SbpSerialize;
 pub trait SBPMessage: SbpSerialize {
     fn get_message_name(&self) -> &'static str;
     fn get_message_type(&self) -> u16;
+    fn message_type() -> u16 where Self: Sized;
```

Comment:
An associated method is needed here so that you can get the message type of a message without having to actually construct one. Not sure the best way to do this. Generally `get_message_type` could be an associated function rather than a method except for in the case of `SBP` where the message type changes depending on the message.

Might be worth having a separate trait like 
```rust
trait OtherSBPMessageTrait {
    fn message_type() -> u16;
}
``` 
that is only implemented by the actual messages

Reply:
And re the `Self: Sized` bound, I think we could probably just add that as 

```
trait SBPMessage: SbpSerialize + Sized {
   ...
}

because all the messages have a known size